Lighting Board/Code/Examples/Lighting_Board/Lighting_Board.ino

```diff
@@ -0,0 +1,152 @@
+#include "LightingRev1.h"
+
+void setup() 
+{
+  Serial.begin(9600);
+
+  //Set up rovecomm with the correct IP and the TCP server
+  RoveComm.begin(RC_MULTIMEDIABOARD_FOURTHOCTET, &TCPServer);
+  delay(100);
+
+  Serial.println("Started Lighing Board");
+    
+
+  //update timekeeping
+  last_update_time = millis();
+
+  
+  autonomy.begin();
+  autonomy.setBrightness(BRIGHTNESS);
+  pinMode(HEADLIGHT_TOGGLE, OUTPUT);
+  digitalWrite(HEADLIGHT_TOGGLE, LOW);
+}
+
+void loop()
+{
+  packet = RoveComm.read();
+  if (packet.data_id != 0)
+  {
+    switch (packet.data_id)
+      {
+        case RC_MULTIMEDIABOARD_HEADLIGHTINTENSITY_DATA_ID:
+        {
+          uint8_t* headlightEnable = (uint8_t*)packet.data;
+          if (headlightEnable > 0)
+            digitalWrite(HEADLIGHT_TOGGLE, HIGH);
+          else
+            digitalWrite(HEADLIGHT_TOGGLE, LOW);
+          break;
+        }
+        case RC_MULTIMEDIABOARD_LEDRGB_DATA_ID:
+        {
+          uint8_t* LEDRGB = (uint8_t*)packet.data;
+          for( uint16_t i=0; i<AUTONOMY_COUNT; i++)
+          {
+            autonomy.setPixelColor(i, autonomy.Color(LEDRGB[0], LEDRGB[1], LEDRGB[2]));
```

Comment:
Maybe use fill()? Might make it easier to understand you are applying the same color to all LEDs.